builtin/providers/aws/resource_aws_rds_cluster_parameter_group_test.go

```diff
@@ -0,0 +1,301 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSDBClusterParameterGroup_basic(t *testing.T) {
```

Comment:
Can you also write a `_disappears` test similar to how I did [here](https://github.com/hashicorp/terraform/pull/5717/files#diff-2511fea892612e5ceee0da1e20ec10d5R37). That way we'll make sure Terraform can still make progress if the remote resource goes away.